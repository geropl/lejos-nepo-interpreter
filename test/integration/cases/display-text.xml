<?xml version="1.0" encoding="UTF-8"?>
<export xmlns="http://de.fhg.iais.roberta.blockly">
    <program>
        <block_set xmlns="http://de.fhg.iais.roberta.blockly" robottype="nxt" xmlversion="3.1">
            <instance x="384" y="50">
                <block type="robControls_start" id="start_1" intask="true" deletable="false">
                    <statement name="ST">
                        <block type="robActions_display_text" id="display_1" intask="true">
                            <value name="OUT">
                                <block type="text" id="text_1" intask="true">
                                    <field name="TEXT">Hello NEPO!</field>
                                </block>
                            </value>
                        </block>
                    </statement>
                </block>
            </instance>
        </block_set>
    </program>
</export>