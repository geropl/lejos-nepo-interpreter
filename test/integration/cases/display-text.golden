clearDisplay()
displayText('Hello NEPO!', 0, 0)
refreshDisplay()
